tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+git+https://github.com/jina-ai/jina.git@v2.0.0rc6#egg=jina[http]
```

Comment:
I think jina is not needed here, as it is installed in the no tests requirements

Reply:
true, just removed it